hngprojects/hng_boilerplate_java_web
# Feature: Comment Deletion Endpoint

### How should this be tested?
* Submit a DELETE request to api/v1/comments/:comment_id without being authenticated.
    * Expected Result: Receive a 401 Unauthorized response.
* Submit a DELETE request with an invalid comment ID (non-existent).
    * Expected Result: Receive a 404 Not Found response.
* Submit a DELETE request with a valid comment ID and a valid bearer token.
    * Expected Result: Receive a 204 OK
### Checklist
* Implemented authentication and authorization checks for the delete endpoint.
* Validated that the comment exists before deletion.
* Implemented soft deletion by setting the delete attribute to True.
* Configured responses for success and error scenarios.
### Reference Issue
[FEAT]: Comment Deletion - Implement Comment Deletion Endpoint
